test_runner/pg_clients/swift/PostgresClientKitExample/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM swift:5.6 AS build
+FROM swift:5.7 AS build
 RUN apt-get -q update && apt-get -q install -y libssl-dev
 WORKDIR /source
 
 COPY . .
 RUN swift build --configuration release
 
-FROM swift:5.6
+FROM swift:5.7
```

Comment:
do we need really `swift` images to execute the binary?

Reply:
Yep, the resulting binary is linked with some of Swift libraries:
```
ldd release/PostgresClientKitExample | grep "/usr/lib/swift"
	libswift_StringProcessing.so => /usr/lib/swift/linux/libswift_StringProcessing.so (0x0000ffff89860000)
	libFoundation.so => /usr/lib/swift/linux/libFoundation.so (0x0000ffff88ff0000)
	libswiftGlibc.so => /usr/lib/swift/linux/libswiftGlibc.so (0x0000ffff88fc0000)
	libswiftDispatch.so => /usr/lib/swift/linux/libswiftDispatch.so (0x0000ffff88ed0000)
	libdispatch.so => /usr/lib/swift/linux/libdispatch.so (0x0000ffff88e60000)
	libBlocksRuntime.so => /usr/lib/swift/linux/libBlocksRuntime.so (0x0000ffff88e40000)
	libswift_Concurrency.so => /usr/lib/swift/linux/libswift_Concurrency.so (0x0000ffff88920000)
	libswiftCore.so => /usr/lib/swift/linux/libswiftCore.so (0x0000ffff88300000)
	libswift_RegexParser.so => /usr/lib/swift/linux/libswift_RegexParser.so (0x0000ffff88030000)
	libicuucswift.so.65 => /usr/lib/swift/linux/libicuucswift.so.65 (0x0000ffff87bd0000)
	libicui18nswift.so.65 => /usr/lib/swift/linux/libicui18nswift.so.65 (0x0000ffff878c0000)
	libicudataswift.so.65 => /usr/lib/swift/linux/libicudataswift.so.65 (0x0000ffff85e00000)
```

Although Swift has added support for [static linking](https://forums.swift.org/t/static-linking-on-linux-in-swift-5-3-1/41989), it didn't work for me out of the box. It fails with the error `error: undefined reference to 'std::__throw_bad_array_new_length()'` and I don't really into debugging it. 
It doesn't really matter for our case if we use Swift container with runtime, so I'll leave it.

Reply:
ok :(